base/server/test/com/netscape/cmscore/password/PlainPasswordFileTest.java

```diff
@@ -0,0 +1,135 @@
+// --- BEGIN COPYRIGHT BLOCK ---
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; version 2 of the License.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+//
+// (C) 2018 Red Hat, Inc.
+// All rights reserved.
+// --- END COPYRIGHT BLOCK ---
+package com.netscape.cmscore.password;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+
+import com.netscape.cmsutil.password.PlainPasswordFile;
+
+/**
+ * @author Dinesh Prasanth M K <dmoluguw@redhat.com>
+ *
+ */
+public class PlainPasswordFileTest {
+
+    PlainPasswordFile pwdFile;
+    File createdFile;
+    private final int TESTCASE_ENTRY = 0;
+    private final int TESTCASE_KEY = 1;
+    private final int TESTCASE_VALUE = 2;
+
+    // Successful cases
+    String[][] testCases = {
+            { "Truth=Beauty", "Truth", "Beauty" }, // No spaces
+            { " Truth = Beauty ", "Truth", "Beauty" }, // Check for spaces
+            { "Welcome Message=Hello World", "Welcome Message", "Hello World" }, // No spaces
+            { " Welcome Message = Hello World ", "Welcome Message", "Hello World" },
+            { "\n" }, // Empty line
```

Comment:
We don't need to include \n here since it will be added by writeToFileAndInit(). So this testcase should be just an empty string "".